pkg/v1/tkg/client/upgrade_cluster.go

```diff
@@ -1068,56 +1064,46 @@ func (c *TkgClient) createVsphereInfrastructureTemplateForUpgrade(regionalCluste
 }
 
 func (c *TkgClient) patchKubernetesVersionToKubeadmControlPlane(regionalClusterClient clusterclient.Client, clusterUpgradeConfig *clusterUpgradeInfo) error {
-	if clusterUpgradeConfig.ActualComponentInfo.KubernetesVersion == clusterUpgradeConfig.UpgradeComponentInfo.KubernetesVersion &&
+
+	log.Infof("Cluster Name: %s, Cluster Namespace %s", clusterUpgradeConfig.ClusterName, clusterUpgradeConfig.ClusterNamespace)
```

Comment:
nit: is this log needed or just added for debugging purposes?

Reply:
It's used for debugging. I'll up the log level.